cli_tools/diagnostics/main_windows.go

```diff
@@ -204,6 +211,36 @@ func collectFilePaths(roots []string) ([]string, []error) {
 	return filePaths, errs
 }
 
+// getCommandToConvertEvt return `Get-WinEvent (-ProviderName/-LogName) ${logName}` powershell cmd string
+// based on log source: provider or log name.
```

Comment:
This comment is a little too detailed. First, there's no need to include the function name in the comment describing the function. Consider something like:

```
Returns a Get-WinEvent command to get the event log for the specified log source.
```